docs/en/modtap.md

```diff
@@ -1,39 +1,39 @@
-# ModTap Keycodes
```

Comment:
This file has to be renamed to `holdtap.md`.

Reply:
I have changed it to `holdtap.md`